File: CS/ReadOnlyTabStopRemover.Module.Win/WinModule.cs
using System;
using System.ComponentModel;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Model;

namespace ReadOnlyTabStopRemover.Module.Win
{
    [ToolboxItemFilter("Xaf.Platform.Win")]
    public sealed partial class ReadOnlyTabStopRemoverWindowsFormsModule : ModuleBase {
        public ReadOnlyTabStopRemoverWindowsFormsModule() {
            InitializeComponent();
        }

        public override void ExtendModelInterfaces(ModelInterfaceExtenders extenders) {
            base.ExtendModelInterfaces(extenders);
            extenders.Add<IModelOptions, IModelOptionsEx>();
        }
    }
    public interface IModelOptionsEx : IModelNode {
        [DefaultValue(false)]
        [Category("Behavior")]
        bool TabOverReadOnlyEditors { get; set; }
    }
}

File: VB/ReadOnlyTabStopRemover.Module.Win/WinModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Model

Namespace ReadOnlyTabStopRemover.Module.Win
	<ToolboxItemFilter("Xaf.Platform.Win")> _
	Public NotInheritable Partial Class ReadOnlyTabStopRemoverWindowsFormsModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub

		Public Overrides Sub ExtendModelInterfaces(ByVal extenders As ModelInterfaceExtenders)
			MyBase.ExtendModelInterfaces(extenders)
			extenders.Add(Of IModelOptions, IModelOptionsEx)()
		End Sub
	End Class
	Public Interface IModelOptionsEx
	Inherits IModelNode
		<DefaultValue(False), Category("Behavior")> _
		Property TabOverReadOnlyEditors() As Boolean
	End Interface
End Namespace
